### PR TITLE
Ditto issue #921: Deprecated HttpStatusCode and replaced it with HttpStatus

### PR DESCRIPTION
**Pull request by jufickel-b**
Signed-off-by: Juergen Fickel <juergen.fickel@bosch.io>

### CODE REVIEW

**Comment by thjaeckle** [comment]
@jufickel-b in order to fix the failing "license-check", could you please do the same adjustment here as was done by @yufei-cai in the Ditto repo:
https://github.com/eclipse/ditto/commit/0379451a9adce00efc3c70890d1d602585d62757

**Comment by DerSchwilk** [comment]
LGTM, build should be working after ditto merge #118

**Review by DerSchwilk — APPROVED** [review]
LGTM

**Pull request merged**
